common/src/main/resources/data/conditional/recipes/farm_and_charm/mincer/cornmeal_from_brewery.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "doapi:conditional",
```

Comment:
DoAPI will most likely be removed at some point + I'm not familiar with its recipe types
I'd appreciate if we used the `forge:mod_loaded` and the `fabric:all_mods_loaded` conditions where possible, to reduce dependencies and make this more readable

For reference, here's the fabric way:
```json
    "fabric:load_conditions": [
        {
            "condition": "fabric:all_mods_loaded",
            "values": [
                "brewery",
            ]
        }
    ]
 ```

Reply:
That works for me, I'll update. Is there a standard for confirming that a specific mod is NOT loaded in a conditional recipe?

Reply:
For fabric, you can use
```json
"fabric:load_conditions": [
    {
      "condition": "fabric:not",
      "value": {
        "condition": "fabric:all_mods_loaded",
        "values": [
          "some-mode"
        ]
      }
    },
    {
      "condition": "fabric:all_mods_loaded",
      "values": [
        "some-mod"
      ]
    }
  ]
  ```

Reply:
And for forge, use
```json
  "type": "forge:not",
  "value": {
      "type": "forge:mod_loaded",
      "modid": "some-mod
  }
  ```